waterbutler/auth/osf/handler.py

```diff
@@ -9,26 +9,18 @@
 
 from waterbutler.auth.osf import settings
 
-
 JWE_KEY = jwe.kdf(settings.JWE_SECRET.encode(), settings.JWE_SALT.encode())
 
 
 class OsfAuthHandler(auth.BaseAuthHandler):
     """Identity lookup via the Open Science Framework"""
     ACTION_MAP = {
         'put': 'upload',
-        'post': 'copyfrom',
```

Comment:
this might break v0, need to test

Reply:
v0 uses auth.fetch() over auth.get(). Fetch does not use the ACTION_MAP dict, so it should be fine.